README.md

```diff
@@ -43,6 +43,41 @@ conda activate dbcp-dev
 
 This conda environment has python, pip and pre-commit installed in it. This env is just for running pre-commits, the actual ETL development happens in docker.
 
+## GCP Authentication
```

Comment:
Got fed up with authenticating everytime we ran the ETL so I finally figured out how to mount local user credentials into the docker container.